packages/sources/peregrine-fund-admin/src/config/index.d.ts

```diff
@@ -0,0 +1,29 @@
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
```

Comment:
How did you generate this?
Under this folder we only need index.ts

Reply:
Thank you, Cleaned folder and removed, was from a pathing error before. I was editing the local tsconfig.json files when I should have been updating the project root files.